importService/importFileParser.js

```diff
@@ -0,0 +1,57 @@
+import csv from "csv-parser";
+import {
+  S3Client,
+  GetObjectCommand,
+  CopyObjectCommand,
+  DeleteObjectCommand,
+} from "@aws-sdk/client-s3";
+import { finished } from "stream/promises";
+
+const client = new S3Client({ region: "eu-west-1" });
+
+export async function importFileParser(event) {
+  try {
+    const bucketName = event.Records[0].s3.bucket.name;
+    const objectKey = decodeURIComponent(
+      event.Records[0].s3.object.key.replace(/\+/g, " "),
```

Comment:
Technically it is possible that we get 'undefined' doing Records[0]. And then getting something from 'undefined' we would see unexpected error. So it make sence to prevent such cases